src/screens/DashboardScreen.tsx

```diff
@@ -69,59 +71,88 @@ const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
       verticalGap
       headerOptions={{
         headerRight: () => <DashboardHeaderActions />,
-        headerLeft: () => <WalletSwitchButton />,
+        headerLeft: () => <WalletSwitchButton isLoading={isLoading} />,
         headerTitle: walletName
       }}
       {...props}
     >
       <BalanceAndButtons>
         <BalanceSummary dateLabel="VALUE TODAY" />
-        <ButtonsRowContainer style={buttonsRowStyle}>
-          <ButtonsRow sticked hasDivider={totalBalance > 0}>
-            {totalBalance > 0 && (
-              <Button
-                onPress={() => navigation.navigate('SendNavigation')}
-                iconProps={{ name: 'arrow-up-outline' }}
-                title="Send"
-                type="transparent"
-                color={theme.global.send}
-                flex
-              />
-            )}
+        {totalBalance > 0 && (
+          <ButtonsRowContainer
+            style={[
+              buttonsRowStyle,
+              {
+                shadowColor: 'black',
+                shadowOffset: { height: 5, width: 0 },
+                shadowOpacity: theme.name === 'dark' ? 0.5 : 0.08,
+                shadowRadius: 3
+              }
+            ]}
+          >
+            <Button
+              onPress={() => navigation.navigate('SendNavigation')}
+              iconProps={{ name: 'arrow-up-outline' }}
+              title="Send"
+              variant="highlightedIcon"
+              round
+              short
+              flex
+            />
             <Button
               onPress={() => navigation.navigate('ReceiveNavigation')}
               iconProps={{ name: 'arrow-down-outline' }}
               title="Receive"
-              type="transparent"
-              color={theme.global.receive}
+              variant="highlightedIcon"
+              round
+              short
               flex
             />
-          </ButtonsRow>
-        </ButtonsRowContainer>
+          </ButtonsRowContainer>
+        )}
       </BalanceAndButtons>
       <AddressesTokensList />
+      {totalBalance === BigInt(0) && (
```

Comment:
`totalBalance === 0` works as well (we have both variants in this component, let's choose one for consistency)